tests/NoFollowUgcAndImageTest.php

```diff
@@ -0,0 +1,95 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\Markdown;
+
+use type Facebook\HackTest\DataProvider;
+use function Facebook\FBExpect\expect;
+use namespace HH\Lib\{Str};
+
+final class NoFollowUgcAndImageTest extends TestCase {
+    protected function assertXSSExampleMatches(
```

Comment:
I'm trying to run "Format Document" in visual studio but not getting any changes to the file

Reply:
Are you running it within a Slack codebase? If not, this is perhaps vscode settings - the default is 2 space rather than 4, but IIRC Slack uses 4 space

Reply:
Ah thanks for the tip! The setting was 4 spaces in my non-Slack codebase, fixed it and updated!